@here/harp-map-controls/lib/MapControls.ts

```diff
@@ -1097,6 +1100,8 @@ export class MapControls extends THREE.EventDispatcher {
     }
 
     private convertTouchPoint(touch: Touch, oldTouchState?: TouchState): TouchState | undefined {
+        // TODO: Test if touch event gives relative to canvas coordinates, this touch
+        // coordinates are sometimes used (double touch) to calculate NDC values.
```

Comment:
This should be similar to mouse events: `.pageX` and `.pageY` are position relative to page. So we could have the same issue if the canvas does not cover the whole page. Switching to `.clientX` and `.clientY` and accounting for `domElement.getBoundingClientRect()` should also be done here.